plugins/view-resources/src/components/EditDoc.svelte

```diff
@@ -256,12 +257,13 @@
         on:open={(ev) => {
           ignoreKeys = ev.detail.ignoreKeys
           ignoreMixins = new Set(ev.detail.ignoreMixins)
+          allowedCollections = ev.detail.allowedCollections ?? []
           getMixins(parentClass, object)
           updateKeys()
         }}
       />
     {/if}
-    {#each collectionEditors as collection}
+    {#each collectionEditors.filter((it) => !allowedCollections.includes(it.key.key)) as collection}
```

Comment:
May be better place for it in `utils/collectionsFilter`?

Reply:
Not sure, let it be for now.